Directory.Packages.props

```diff
@@ -0,0 +1,32 @@
+<Project>
+
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="[1.0.0, 2.0)" />
+    <PackageVersion Include="System.Collections.Immutable" Version="1.7.1" />
+    <PackageVersion Include="System.Threading.Channels" Version="6.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="AutoFixture" Version="4.17.0" />
+    <PackageVersion Include="BenchmarkDotNet" Version="0.13.1" />
+    <PackageVersion Include="coverlet.collector" Version="3.1.2" />
+    <PackageVersion Include="coverlet.msbuild" Version="3.1.2" />
+    <PackageVersion Include="FluentAssertions" Version="6.7.0" />
+    <PackageVersion Include="GitHubActionsTestLogger" Version="2.3.3" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
+    <PackageVersion Include="NSubstitute" Version="5.0.0" />
+    <PackageVersion Include="NUnit" Version="3.13.2" />
+    <PackageVersion Include="OpenFeature.Contrib.Providers.Flagd" Version="0.1.5" />
+    <PackageVersion Include="SpecFlow" Version="3.9.74" />
+    <PackageVersion Include="SpecFlow.Tools.MsBuild.Generation" Version="3.9.74" />
+    <PackageVersion Include="SpecFlow.xUnit" Version="3.9.74" />
+    <PackageVersion Include="xunit" Version="[2.4.1, 3.0)" />
+    <PackageVersion Include="xunit.runner.visualstudio" Version="[2.4.3, 3.0)" />
```

Comment:
Is it even worth while have a range on xunit, we should just target the latest version as its a testing dependencies

Reply:
IMO absolutely not, would love to just set these like normal. I was actually going to sneak that in here, but the latest version of xUnit has some new analyzers that lit up when I did that, so pulled back to minimize the excitement in this PR.

If you're onboard with it, though, I'll take this^ on as a follow up task once this lands.

Reply:
Yea we can resolve those changes under another PR